.circleci/config.yml

```diff
@@ -612,6 +612,11 @@ jobs:
           command: |
             just snapshots-check || echo "export SNAPSHOTS_STATUS=1" >> "$BASH_ENV"
           working_directory: packages/contracts-bedrock
+      - run:
+          name: interfaces
+          command: |
+            just interfaces-check || echo "export INTERFACES_STATUS=1" >> "$BASH_ENV"
+          working_directory: packages/contracts-bedrock
```

Comment:
Out of scope for this PR, but I never liked this `echo "export INTERFACES_STATUS=1" >> "$BASH_ENV"` pattern, and contributors less familiar with the codebase sometimes seem confused by it and are unsure what exactly failed.

I'd prefer to just fail the step directly, and change the workflow settings so all checks run even when previous ones fail

Reply:
Probably worth opening an issue for this